### ISSUE TITLE
How to handle video data with an existing image classification network

### ISSUE THREAD

**Issue by nicklhy** [apache/incubator-mxnet#2869]
I am trying to build a network that can process several video frames and output a class label with a softmax function. The baseline is like this: Frames from the same video are first processed by a normal CNN(i.e. inception-bn). Then, they are fused by a temporal pooling layer which generates a single feature representation for a video. A soft-max can be added after that to predict the label I need.
The problem is the data inputs of those existing imagenet pre-trained model(saved as json files) are already fixed. If I want to feed 10 video files(batch size), 15 frames per video, to the network each time. There are usually two different solutions:
a) The shapes of the input data and label are set as (10, 15, 3, height, width) and (10,) respectively. Then I need to reshape the data to (150, 3, height, width) and extract feature for each frame with the inception-bn symbol. Before the temporal pooling layer, a reverse reshape layer is also added to transform the features to (10, 15, feat-dim) again. But I do not know how to replace the original data symbol with a data->reshape symbol.
b) The shapes of the input data and label are set as (150, 3, height, width) and (10, ) respectively. Thus, each frame can be processed by the inception-bn separately. But there is another problem, since the first dims of data shape and label shape are different, neither batch-size=150 nor batch-size=15 can really work during the training phase(the data parallel mechanism would slice the data and label based on the data batch-size).

Can any one come and provide some help about it?

**Comment by starimpact** [comment]
mxnet can proceed fixed different size input.that means it can use a new size input which is different from the size of training.

**Comment by phunterlau** [comment]
This issue is closed due to lack of activity in the last 90 days. Feel free to reopen if this is still an active issue. Thanks!

**Comment by guruvishnuvardan** [comment]
HI Can you please help me with error, 
![mxnet data_shape](https://user-images.githubusercontent.com/31349523/36965592-078c35c6-2052-11e8-843c-86c67387b0dd.png)

Please find my code here: https://github.com/guruvishnuvardan/MXnet-SSD---Object-Detection/blob/master/demo.py

Is batch size cause of the defect?, if so, Can you please help define batch size for a 5 min mp4 video or a live web cam?
